src/Microsoft.ML.CodeGenerator/CodeGenerator/CSharp/CodeGenerator.cs

```diff
@@ -257,77 +257,12 @@ internal IList<(string, string[])> GenerateTransformsAndUsings(IEnumerable<Pipel
             return (trainerString, trainerUsings);
         }
 
+        /// <summary>
+        /// Utilize <see cref="ML.CodeGenerator.Utilities.Utils.GenerateClassLabels(ColumnInferenceResults, IDictionary{string, CodeGeneratorSettings.ColumnMapping})"/>
+        /// </summary>
         internal IList<string> GenerateClassLabels(IDictionary<string, CodeGeneratorSettings.ColumnMapping> columnMapping = default)
```

Comment:
I've decided to keep this `GenerateClassLabels` as it is possible that a `CodeGenerator` instance can call this function, as it does below: